### PR TITLE
 medical attention REQuired

### PR DESCRIPTION
**Pull request by fernandos13**
## About The Pull Request
this allow req to "sell/require/buy" some medical items
being:

**name = "Advanced medical packs"**
_5 brute kits 5 burn kits 5 splints
cost: 10_

**name = "tweezers"**
_contains a pair of tweezers
cost: 20_

**name = "advanced big hypospray"**
_contains a big hypo spray aka 120u
cost: 12_    ((regular hypo is 5 so this is a lil more than double the price))

## Why It's Good For The Game
adds a way to ressuply medics on the field with a more direct interaction from req instead of needing MOS to restock and vend kits. also gives ways to re aquire lost tweezers and big hypos although for a steep price

## Changelog
:cl:
add: big hypo to the req orders
add: advanced medical pack crate to the req orders
add: tweezers to the req orders
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -6. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by fernandos13** [comment]
also i have no idea whats causing the errors since i din't even touched the overlay part of the code

**Comment by fernandos13** [comment]
also getting a error on:
`/datum/supply_packs/medical/tweezers
	name = "tweezers"
	notes = "contains a pair of tweezers."
	contains = list(/obj/item/tweezers)
	cost = 20  //shouldn't be easy to get
	containertype = /obj/structure/closet/crate/secure/surgery
	access = ACCESS_MARINE_MEDBAY`
	
	for the `error: /obj/item/tweezers: undefined type path` and i'm not getting why its happening

File: code/modules/reqs/supplypacks.dm [hunk 1]
```diff
@@ -1031,6 +1038,38 @@ MEDICAL
...
+		/obj/item/stack/medical/advanced/ointment,
+		/obj/item/stack/medical/advanced/ointment,
+		/obj/item/stack/medical/advanced/ointment,
+		/obj/item/stack/medical/advanced/ointment,
+		/obj/item/stack/medical/splint,
+		/obj/item/stack/medical/splint,
+		/obj/item/stack/medical/splint,
+		/obj/item/stack/medical/splint,
+		/obj/item/stack/medical/splint,
+		)
```
Comment (Chaznoodles) [inline code/modules/reqs/supplypacks.dm:1060]: Move the close bracket on 1060 to the end of line 1059 to solve the inconsistent indentation problem that's causing it to fail.

Comment (fernandos13) [reply code/modules/reqs/supplypacks.dm:1060]: OH fuck that was a stupid mistake

**Comment by Losenis** [comment]
I'd sooner suggest just adding tweezers into the contents of the already existing surgery crate, given shrapnel removal falls under the umbrella of surgery, even if ghetto.

**Comment by BraveMole** [comment]
code/modules/reqs/supplypacks.dm:1044:	contains = list(
		/obj/item/stack/medical/advanced/bruise_pack,
		/obj/item/stack/medical/advanced/bruise_pack,
		/obj/item/stack/medical/advanced/bruise_pack,
		/obj/item/stack/medical/advanced/bruise_pack,
		/obj/item/stack/medical/advanced/bruise_pack,
		/obj/item/stack/medical/advanced/ointment,
		/obj/item/stack/medical/advanced/ointment,
		/obj/item/stack/medical/advanced/ointment,
		/obj/item/stack/medical/advanced/ointment,
		/obj/item/stack/medical/advanced/ointment,
		/obj/item/stack/medical/splint,
		/obj/item/stack/medical/splint,
		/obj/item/stack/medical/splint,
		/obj/item/stack/medical/splint,
		/obj/item/stack/medical/splint,)
the ) in a long list should match identation of the opening list line

This is what is causing the error.  The last ) needs to go to another line, be sure to copy how it's done for the other packs

**Review by SplinterGP — APPROVED** [review]
this is fine

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-21). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**